app/views/application/_index_actions.html.slim

```diff
@@ -1,10 +1,5 @@
-- target_controller ||= controller_name
-
-td= link_to t_action(:show), subject
-td= link_to t_action(:print), { controller: target_controller, action: :show, id: subject.id, print: 'true' }, target: '_blank'
-- if ['clients', 'volunteers'].include? target_controller
-  td= link_to t_model(Journal), { controller: 'journals', action: 'index', "#{target_controller.singularize}_id": subject.id }
-- if policy(subject.class).index?
-  td= link_to t_action(:edit), { controller: target_controller, action: :edit, id: subject.id }
-- if policy(subject.class).destroy?
-  td= link_to t_action(:destroy), subject, confirm_deleting(subject)
+td.index-action-cell
+  span= link_to t_action(:show), subject
+  span= link_to t_action(:edit), edit_polymorphic_path(subject)
+  - if policy(subject.class).destroy?
+    span= link_to t_action(:destroy), subject, confirm_deleting(subject)
```

Comment:
The `target_controller ||= controller_name` was already removed once and then added back to fix a bug, are you sure you're not introducing that bug again? It's still referenced in `app/views/assignments/_assignment.html.slim`:

```slim
= render 'index_actions', subject: assignment, target_controller: 'assignments'
```

Reply:
See bugfix from @chrysalag in 9f01cf9d

Reply:
but I think using polymorphic path would make use of that controller_name or similar obsolete ... or wouldn't it?

Reply:
The thing is, that problem was mainly because the actions `assignments#find_volunteer` and  `assignments#find_client` where woven tightly into the client and volunteers views. I just untangled this with clients for now, and Intend to to the same with Volunteers views ASAP, because those views are unreadable and generally just error prone.

Reply:
Hmm not sure, the test from the bugfix still seems to pass so I guess it's fine? :)